src/histolab/tiler.py

```diff
@@ -28,8 +28,8 @@ class Tiler(Protocol):
     tile_size: int
 
     @lru_cache(maxsize=100)
-    def box_mask(self, slide: Slide) -> sparse._coo.core.COO:
-        """Return binary mask at level 0 of the box to consider for tiles extraction.
+    def box_mask_thumb(self, slide: Slide) -> np.ndarray:
+        """Return binary mask at thumbnail dimensions of the box for tiles extraction.
```

Comment:
""Return binary mask at thumbnail dimensions of the box for tiles extraction." --> "Return binary mask, at thumbnail level, of the box for tiles extraction.